release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2764,3 +2764,55 @@ This issue is resolved for:
 
 To upgrade an existing {product-title} 4.5 cluster to this latest release, see
 xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI for instructions.]
+
+[id="ocp-4-5-15"]
+=== RHBA-2020:4228 - {product-title} 4.5.15 bug fix update
+
+Issued: 2020-10-19
+
+{product-title} release 4.5.15 is now available. The list of container images
+and bug fixes included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:4228[RHBA-2020:4228] advisory.
+The RPM packages included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:4229[RHBA-2020:4229] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5493561[{product-title} 4.5.15 container image list]
+
+[id="ocp-4-5-15-bug-fixes"]
+==== Bug fixes
+
+* A new API version for CSR was introduced in {product-title} 4.6, and as a consequence, older versions could not approve or deny certificates during upgrade. With this release, versions of CSR in older versions of `oc` are tolerated so that it is possible to deny or approve certificates with `oc` 4.5 against {product-title} server 4.6.
```

Comment:
Should `{product-title} 4.6` be `{product-title} _server_ 4.6`_ in the first sentence? Also, seems odd that this references OCP 4.6 before 4.6 GA, but I admittedly know little about this bug fix.